spec/collections/samples/sample_ontology_link.yaml

```diff
@@ -0,0 +1,47 @@
+name: sample_ontology_link
+type: edge
+schema:
+  "$schema": http://json-schema.org/draft-07/schema#
+  type: object
+  required: [_from, _to, created]
+  description: The _from sample node is associated with the _to taxon.
+  properties:
+    _from:
+      type: string
+      description: the sample node _id (as found in sample_nodes collection)
+    _to:
+      type: string
+      description: The _id of an ontology vertex, such as from ENVO, GO, etc.
+    created:
+      type: integer
+      description: timestamp of when the link was created
```

Comment:
Timestamp in what format?

Reply:
Probably should be edited to say "unix epoch" (i assume since it's an integer)

Reply:
I would also put "minimum: 0", cause why not